wcivf/apps/elections/views/election_views.py

```diff
@@ -22,14 +21,10 @@ class ElectionsView(TemplateView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         Election = apps.get_model("elections.Election")
-        all_elections = Election.objects.all().order_by(
-            "-election_date", "election_type", "name"
-        )
-
-        context["past_elections"] = all_elections.past()
-        context[
-            "current_or_future_elections"
-        ] = all_elections.current_or_future()
+        qs = Election.objects.all().order_by("-election_date", "name")
```

Comment:
You might want to keep `election_type` in the `order_by` as before, so that on the unfiltered page, the elections are grouped by election type?